app/utils/dayjs/configureDayjs.ts

```diff
@@ -9,12 +9,18 @@ import advancedFormat from 'dayjs/plugin/advancedFormat';
 import localizedFormat from 'dayjs/plugin/localizedFormat';
 import objectSupport from 'dayjs/plugin/objectSupport';
 
-import { DEFAULT_LOCALE } from 'i18n';
+import { DEFAULT_LOCALE, appLocales } from 'i18n';
 
 import { CustomLocalesConfig, RELATIVE_TIME_OPTIONS } from './constants';
 
-// Add imports for all new app languages here
-require('dayjs/locale/ar');
+appLocales.forEach((locale) => {
+  try {
+    // eslint-disable-next-line global-require
+    require(`dayjs/locale/${locale}.js`);
+  } catch (e) {
+    console.error(e);
```

Comment:
Is this console.error necessary?

Reply:
Not all languages added to the app can be supported by dayjs. It was intended to inform developer about not supported languages. I removed that message and [added a info to the code and docs instead](https://github.com/Michigan-State-University/cias-web/pull/202/commits/adaf2e5d20dc8402bdcd8bcede5407aede5f99c4). Do you think that's fine now?